lib/lol/request.rb

```diff
@@ -129,7 +129,7 @@ def perform_request url, verb = :get, body = nil, options = {}
     def perform_rate_limited_request url, verb = :get, body = nil, options = {}
       return perform_uncached_request(url, verb, body, options) unless rate_limiter
       @rate_limiter.times 1 do
-        perform_uncached_request(url, verb, body, options)
+        return perform_uncached_request(url, verb, body, options)
```

Comment:
Even if executed only 1 time I think that is not safe use a return in a block like this. 

What you can do is:
-  Initialize a variable outside of the block
- assign the value returned from `perform_uncached_request` to that variable
- return the variable outside of the block

@nicolaracco what do you think?

Reply:
I was thinking the same thing, but I didn't read into how each of the responses were formatted (e.g. a single champion vs a list of champions) so I left it as is.

Reply:
@oeN Why do you think this isn't safe?

Reply:
IMO I think a return here is fine, I don't see a reason why you would ever want to run the block twice, so I think the point about if it were run 2 times to be moot. Even if there was a reason to change it from 1 times to 2 times, then I think it would be better to have that change be made at the same time. I think your needlessly worrying about future-proofing.

Reply:
@oeN any thoughts/updates?

Reply:
@hebron-george I'll merge and bump the version ASAP